sqlstore/src/main/resources/db/migration/V1700__auto_main.sql

```diff
@@ -0,0 +1,3 @@
+-- update_url_field
```

Comment:
New migrations need to be added in the V8000-8999 range and should have a descriptive name. The "auto_main" migrations are generated from these during release